### ISSUE TITLE
Cosmos DB with Mongo API requesing update right after creation

### ISSUE THREAD

**Issue by epomatti** [terraform-providers/terraform-provider-azurerm#7401]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

Terraform version: 0.12.26
AzureRM version: I'm using the default:

```
provider "azurerm" {
  features {
  }
}
```

### Affected Resource(s)

* `azurerm_cosmosdb_account`

### Terraform Configuration Files

```hcl
resource "azurerm_cosmosdb_account" "default" {
  name                = local.env.cosmos_account_name
  location            = local.env.location
  resource_group_name = local.env.resource_group_name
  offer_type          = "Standard"
  kind                = "MongoDB"

  enable_automatic_failover = false

  consistency_policy {
    consistency_level = "Session"
  }

  geo_location {
    location          = local.env.location
    failover_priority = 0
  }

  capabilities {
    name              = "EnableMongo"
  }

  lifecycle {
    prevent_destroy = true
  }

  tags = local.tags

}
```

### Expected Behavior

Cosmos DB with Mongo should be created with the latest version.

### Actual Behavior

I can see that the server version is 3.6:

![image](https://user-images.githubusercontent.com/6530769/85055996-dbca1c80-b174-11ea-8625-93be98d1621f.png)

But I'm getting the following warning.

I would expect the Mongo DB to be with the latest available at Azure.

> Your Azure Cosmos DB's API for MongoDB account is eligible to be upgraded to the latest Mongo version without downtime. 

![image](https://user-images.githubusercontent.com/6530769/85056095-0025f900-b175-11ea-9fbd-f0093e5543e3.png)

### Steps to Reproduce

1. Create a Cosmos DB account with Mongo capability:

```
  capabilities {
    name              = "EnableMongo"
  }
```

Should create Mongo with the lastest Azure-provided runtime.

**Comment by yupwei68** [comment]
Hi @epomatti , thanks for opening this issue. Sorry for the delay. Currently we have upgraded our cosmos service api version to `2020-04-01` and released in `2.20.0`. Please refer to https://github.com/terraform-providers/terraform-provider-azurerm/pull/7597.
I have tested your configuration and the notification doesn't show up in `2.20.0`. Please try our new release.

**Comment by epomatti** [comment]
Hi @yupwei68 thanks again.

Yes, I tried it and the Portal did not ask for an update. Thanks!
